INSTALL.md

```diff
@@ -1,17 +1,55 @@
-# Install Zync (only explained for MacOS so far)
-1. Download this repository.
+# Install Zync
+
+## Download this repository.
 `git clone git@github.com:3scale/zync.git`
 
-2. Move to the folder of the project.
-`cd zync`
+## Setup PostgreSQL on Mac
+
+There is `Brewfile` containing all the dependencies.
+
+```
+cd zync
+brew bundle
+brew services start postgresql
+```
+
+The command `brew services start postgresql` starts the service of PostgreSQL. If later `./bin/setup` aborts, make sure that the `PostgreSQL` service is running. Verify with `brew services list` that has a status `started` and looking green. If the status is `started` but coloured orange, fix the errors indicated in the log located in `/usr/local/var/log/postgres.log`
```

Comment:
This is more documenting `brew` than zync project

Reply:
This comes from original text. Shall I remove it?

Reply:
Yes I think you should remove it. Lets try to focus on this README to make it so that people know about the project.
Then we can probably add some other `doc/*.md` files or just wiki page for some use cases.